packages/material-ui-lab/src/ToggleButton/ToggleButton.js

```diff
@@ -157,6 +173,10 @@ ToggleButton.propTypes = {
    * If `true`, the button will be rendered in an active state.
    */
   selected: PropTypes.bool,
+  /**
+   * The size of the button.
```

Comment:
```suggestion
   * @ignore
```

The prop only needs to be applied in `ToggleButtonGroup`.